src/components/Dialogs/FittingDialog/FittingDialogComponent.tsx

```diff
@@ -1,14 +1,15 @@
 import * as React from "react";
 import {observer} from "mobx-react";
 import {action, makeObservable, observable} from "mobx";
-import {AnchorButton, Classes, Dialog, FormGroup, HTMLSelect, Icon, IDialogProps, Intent, NonIdealState, Position, Pre, Slider, Tab, Tabs, Text} from "@blueprintjs/core";
+import {AnchorButton, ButtonGroup, Classes, Dialog, FormGroup, HTMLSelect, Icon, IDialogProps, Intent, NonIdealState, Position, Pre, Slider, Tab, Tabs, Text} from "@blueprintjs/core";
 import {Tooltip2} from "@blueprintjs/popover2";
 import classNames from "classnames";
 import {AppStore, HelpType} from "stores";
 import {DraggableDialogComponent} from "components/Dialogs";
 import {SafeNumericInput} from "components/Shared";
 import {CustomIcon} from "icons/CustomIcons";
 import "./FittingDialogComponent.scss";
+import {exportTxtFile, getTimestamp} from "utilities";
```

Comment:
Please move to the line before `import "./FittingDialogComponent.scss";`

Reply:
Done
![image](https://user-images.githubusercontent.com/106953609/193226130-4f5a43d3-caf4-47d8-a239-132ed4476207.png)